refmt_test/extensions.t/run.t

```diff
@@ -12,7 +12,7 @@ Format extensions
   
   type a = [%extend int];
   
-  let%extend x = "hi";
+  [%%extend let x = "hi"];
```

Comment:
This is expected since it's a top_level extension with a let binding. I can fix this particular case or leave as it is now not entirely sure if it's worth it.